.github/workflows/elixir.yml

```diff
@@ -2,16 +2,16 @@ name: Elixir CI
 
 on:
   push:
-    branches: [ main ]
+    branches: [main]
   pull_request:
   workflow_dispatch:
 
 jobs:
   test:
     uses: mtrudel/elixir-ci-actions/.github/workflows/test.yml@main
     with:
-      apt-packages: "libmnl-dev libnl-3-dev"
+      apt-packages: ""
```

Comment:
You can get rid of this entire 'with' stanza here (I can't use the suggestion feature for it since the with line isn't in the diff).